cms/static/sass/views/_bulksettings.scss

```diff
@@ -71,6 +47,27 @@
         tr td {
             padding: 5px;
         }
+
+        .subsection-header {
```

Comment:
The CSS diff for text wrap looks like a massive block of changes, instead of the small series of swaps that actually happened.  Should I split this commit into 2+ commits to show this more clearly, or is it still understandable?